### PR TITLE
Simple todos graphql

### PR DESCRIPTION
**Pull request by denihs**
New tutorial showing how to use GraphQL with Meteor.

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -28,6 +28,13 @@ sidebar_categories:
     - simple-todos/11-testing
     - simple-todos/12-deploying
     - simple-todos/13-next-steps
+  Simple Todos with GraphQL:
+    - simple-todos-graphql/index
+    - simple-todos-graphql/01-graphql-setup
+    - simple-todos-graphql/02-especific-data
```
Comment (StorytellerCZ) [inline _config.yml:34]: `especific` typo?

File: src/simple-todos-graphql/step01/.meteor/release [hunk 2]
```diff
@@ -0,0 +1 @@
+METEOR@2.0
```
Comment (StorytellerCZ) [inline src/simple-todos-graphql/step01/.meteor/release:1]: Best to upgrade to the latest version.

File: src/simple-todos-graphql/step01/client/main.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Meteor } from 'meteor/meteor';
+import { render } from 'react-dom';
+import { App } from '/imports/ui/App';
+
+import { startGraphQLClient } from 'meteor/quave:graphql/client';
```
Comment (StorytellerCZ) [inline src/simple-todos-graphql/step01/client/main.jsx:6]: As much as `quave:graphql` is fine, I think we should stick to the official one via skeleton.

**Review by StorytellerCZ — CHANGES_REQUESTED** [review]

File: _config.yml (hunk 1 above)
Comment (filipenevola) [reply _config.yml]: fixed

**Review by filipenevola — APPROVED** [review]

**Pull request merged**